documentation/how-to/api-discovery.md

```diff
@@ -170,54 +174,62 @@ the descriptive metadata in the Dataset:
       ```
     
     2. Publish the Product: `gwa apply -i <product.yaml>`
-    
 
 === "Web UI"
     #### Create a Product
 
-    1. Navigate to **Namespaces** -> **Products**.
+    1. Navigate to **Gateways**, select your **Gateway** from the list, then 
+    click the **Products** card.
     2. Click **New Product** in the top right.
    
-    #### Link a Dataset
+    #### Link a BC Data Catalogue Dataset
```

Comment:
Confusingly, no - you can actually also link to datasets not in the BCDC despite what the Portal UI shows.

I had noted in https://github.com/bcgov/api-services-portal/issues/1062

> Also a good opportunity to review the text on this menu - it should instead be titled more like 'Link to a Dataset' and then mention that this includes BCDC datasets.

Feel free to open a distinct issue for just the text and propose an enhancement and we can fix this!